pkg/gatewayserver/io/ws/ws.go

```diff
@@ -82,7 +82,7 @@ func New(ctx context.Context, server io.Server, formatter Formatter, cfg Config)
 		return nil, err
 	}
 
-	router := web.RootRouter()
+	router := web.Router()
```

Comment:
I don't know if we can just do this, it may break in Enterprise.

Reply:
Indeed - API endpoints do not need this kind of rendering.